pkg/kubernetes/scripts/projects.js

```diff
@@ -360,24 +505,83 @@
                     },
                 }).result;
             }
-
+            function createGroup() {
+                return $modal.open({
+                    controller: 'GroupNewCtrl',
+                    templateUrl: 'views/add-group-dialog.html',
+                });                    
+            }
             function createUser() {
                 return $modal.open({
                     controller: 'UserNewCtrl',
                     templateUrl: 'views/add-user-dialog.html',
                 });
             }
-            function createGroup() {
+            function modifyUser(userObj) {
                 return $modal.open({
-                    controller: 'GroupNewCtrl',
-                    templateUrl: 'views/add-group-dialog.html',
+                    animation: false,
+                    controller: 'UserChangeCtrl',
+                    templateUrl: 'views/user-modify.html',
+                    resolve: {
+                        fields: function() {
+                            var fields = {};
+                            fields.user = userObj;
+                            return fields;
+                        }
+                    },
+                }).result;
+            }
+            function addMemberToParent(memberObj) {
+                return $modal.open({
+                    controller: 'UserChangeCtrl',
+                    templateUrl: 'views/user-add-membership-dialog.html',
+                    resolve:{
+                        fields: function() {
+                            var fields = {};
```

Comment:
Lets just combine this inot the initializer above var fields = { memberObj: memberObj ... }